app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("NotVaccinatedError")
+
+        if datetime.date.today() > visitor["vaccine"].get("expiration_date"):
+            raise OutdatedVaccineError("OutdatedVaccineError")
+
+        if visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError("NotWearingMaskError")

```

Comment:
Use more descriptive messages for errors, also you can add the visitor's name to them